docs/content/guides/rows/row-sorting.md

```diff
@@ -453,163 +1238,919 @@ import 'handsontable/dist/handsontable.full.min.css';
+          sellDate: '03/05/2020',
+          sellTime: '88-2768',
+          inStock: false,
+        },
+        {
+          brand: 'Meeveo',
+          model: 'Full-Finger Gloves',
+          price: 314,
+          sellDate: '27/03/2020',
+          sellTime: '51-6775',
+          inStock: true,
+        },
+        {
+          brand: 'Buzzdog',
+          model: 'Mountain Frame',
+          price: 268,
+          sellDate: '29/08/2020',
+          sellTime: '44-4028',
+          inStock: false,
+        },
+        {
+          brand: 'Katz',
+          model: 'Half-Finger Gloves',
+          price: 204,
+          sellDate: '02/10/2020',
+          sellTime: '08-2758',
+          inStock: true,
+        },
+      ]}
+        columns={[
+          {
+            type: 'text',
+            data: 'brand',
+          },
+          {
+            type: 'text',
+            data: 'model',
+          },
+          {
+            type: 'numeric',
+            data: 'price',
+            numericFormat: {
+              pattern: '$ 0,0.00',
+              culture: 'en-US'
+            },
+          },
+          {
+            type: 'date',
+            data: 'sellDate',
+          },
+          {
+            type: 'text',
+            data: 'sellTime',
+          },
+          {
+            type: 'checkbox',
+            data: 'inStock',
+          },
+        ]}
+        colHeaders={[
+          'Brand',
+          'Model',
+          'Price',
+          'Date',
+          'Time',
+          'In stock',
+        ]}
+        columnSorting={{
+          // at initialization, sort rows by column 1, in descending order
+          initialConfig: {
+            column: 1,
+            sortOrder: 'desc',
+          },
+        }}
+        rowHeaders={true}
+        height="auto"
+        width="auto"
+        licenseKey="non-commercial-and-evaluation"
+      />
+      <div className="controls">
+        <br></br>
+        <button onClick={sort_asc}>Sort column 1, in ascending order</button>
+        <button onClick={sort_desc}>Sort column 1, in descending order</button>
+      </div>
+    </>
+  );
+}
+
+/* start:skip-in-preview */
+ReactDOM.render(<YourHandsontableComponent />, document.getElementById('example6'));
+/* end:skip-in-preview */
+```
+
+:::
 
-::: tip
-Hooks are also run when you use the [`clearSort`](@/api/columnSorting.md#clearsort) method or provide a configuration that won't be processed, causing validation to fail.
 :::
 
-## Plugin API
+## Related resources
 
-List of methods exposed by the plugin:
+Read our [blog article](https://handsontable.com/blog/articles/2018/11/feature-spotlight-multi-column-sorting) about multi-column rows sorting.
 
-* [`clearSort`](@/api/columnSorting.md#clearsort)
-* [`getSortConfig`](@/api/columnSorting.md#getsortconfig)
-* [`isSorted`](@/api/columnSorting.md#issorted)
-* [`setSortConfig`](@/api/columnSorting.md#setsortconfig)
-* [`sort`](@/api/columnSorting.md#sort)
+### More examples
```

Comment:
We can add this section later on. We need to determine the default playground used here first.